source/usage-examples/count.txt

```diff
@@ -81,27 +79,27 @@ The following example estimates the number of documents in the
 ``movies`` collection in the ``sample_mflix`` database, and then returns
```

Comment:
on line 73 above i think it should be `Long` rather than `long` b/c that's the name of the kotlin data type https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-long/

Reply:
Updated